pkg/compute/init.go

```diff
@@ -75,19 +127,143 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	c.path = abspath
 
 	if c.name == "" {
-		name := filepath.Base(c.path)
-		fmt.Fprintf(progress, "--name not specified, using %s\n", name)
-		c.name = name
+		c.name = filepath.Base(c.path)
+		fmt.Fprintf(progress, "--name not specified, using %s\n\n", c.name)
+
+		name, err := text.Input(out, fmt.Sprintf("Name: [%s] ", c.name), in)
+		if err != nil {
+			return fmt.Errorf("error reading input: %w", err)
+		}
+		if name != "" {
+			c.name = name
+		}
 	}
 
+	if c.description == "" {
+		c.description, err = text.Input(out, "Description: ", in)
+		if err != nil {
+			return fmt.Errorf("error reading input: %w", err)
+		}
+	}
+
+	if c.author == "" {
+		var defaultEmail string
+		if email := c.Globals.File.Email; email != "" {
+			defaultEmail = fmt.Sprintf(" [%s]", email)
+		}
+
+		c.author, err = text.Input(out, fmt.Sprintf("Author:%s ", defaultEmail), in)
+		if err != nil {
+			return fmt.Errorf("error reading input %w", err)
+		}
+		if c.author == "" {
+			c.author = defaultEmail
+		}
+	}
+
+	if c.from == "" {
+		text.Output(out, "%s", text.Bold("Template:"))
+		for i, template := range defaultTemplates {
+			text.Output(out, "[%d] %s", i, template.Name)
```

Comment:
IMO it would be nice to put the URL of the template in parenthesis here, so that the " . . . or type URL" portion of the prompt is visually reinforced.

Reply:
Added in https://github.com/fastly/cli/pull/5/commits/78fd8c262bf2a19bd64543fc003bdbbff37d04b7